scripts/lib/mkcloud-common.sh

```diff
@@ -140,7 +140,7 @@ function handle_service_dependencies
 
 function determine_mtu
 {
-    LC_ALL=C sort -n /sys/class/net/*/mtu | head -n 1
+    LC_ALL=C sort -n /sys/class/net/!(lo)/mtu | head -n 1
```

Comment:
move the extglob in brackets here instead:

eg:
```bash
( shopt -s extglob ; LC_ALL=C sort -n /sys/class/net/!(lo)/mtu | head -n 1 )
```

Reply:
this is good for several reasons:
a) mkcloud-common code can be used later in qa_crowbarsetup and then it would fail without extglob
b) it avoids side-effects in other code
c) it keeps code together that belongs together

Reply:
done